development/includes/sm-share-sidebar.pug

```diff
@@ -0,0 +1,21 @@
+script(src="/resources/js/sm-share-sidebar.js")
+div.sm-share-sidebar
+  a.share-slider-toggle Share me!
+    i(class="fa fa-heart-o")
+  ul.sm-share-buttons
+    //- <!-- Facebook -->
+    a.sm-share-selector(id="facebook-share-button" href="https://www.facebook.com/sharer/sharer.php?u=$PAGE_URL" target="_blank")
```

Comment:
Looks like you forgot to update this. Please put the anchor tag `<a>` inside of the list element tag `<li>` for all of the items in this list.

If you want to be fancy you could do the same loop and data object you did for the blog side nav.